Podfile

```diff
@@ -32,6 +32,7 @@ target 'TCAT' do
     pod 'Pulley', '~> 2.6'
     pod 'Presentation', :git=> 'https://github.com/mattbarker016/Presentation.git'
     pod 'SnapKit'
+    pod 'WhatsNewKit', '~> 1.0.0'
```

Comment:
I wasn't able to build on Xcode 10 / Swift 4.2 on this version. Use `'~> 1.1.2'` for now, the latest version to support Swift 4.2 properly.